documents/autoupdate.md

```diff
@@ -39,8 +39,14 @@ To add an `npm` hook to a library, update the `package.json` with configuration
 * `npmName` should map to the name of the library on `npm`
 * `npmFileMap` is a list of files to take from the `npm` tarball and host on cdnjs
 * `basePath` will be ignored when copying over to the CDN
-* `files` is a pattern matcher allowing selection of multiple files
+* `files` is a pattern matcher allowing selection of multiple files, see [node-glob](https://github.com/isaacs/node-glob#glob-primer)
+  * you can use `!(...)` to exclude the files which are not needed. For example, if you want to exclude `*.flow` and `*.scss`, you can use:
 
+  ```js
+  "files": [
+    "**/!(*.+(flow|scss))"  //To fetch all files except *.flow and *.scss
```

Comment:
So all the libraries need this single super rule? We are writing examples for as the contributors' reference, I have no idea why you just touched the part without any discussion.